x-pack/test/functional/services/ml/data_visualizer_table.ts

```diff
@@ -133,17 +153,119 @@ export function MachineLearningDataVisualizerTableProvider(
       );
     }
 
-    public async assertViewInLensActionEnabled(fieldName: string) {
+    public async ensureActionsMenuOpen(fieldName: string) {
+      await retry.tryForTime(30 * 1000, async () => {
+        const panel = await find.existsByCssSelector('euiContextMenuPanel');
```

Comment:
Since we don't have much details to check the actions menu popover, we can't be sure at this point if the `euiContextMenuPanel` is an opened action menu (or some other menu popover) nor do we know if it's the popover for the correct table row. So I'd suggest to make sure that all popovers are closed first. Maybe with something like this?
```
public async ensureAllMenuPopoversClosed() {
    await retry.tryForTime(5000, async () => {
        await browser.pressKeys(browser.keys.ESCAPE);
        const popoverExists = await find.existsByCssSelector('euiContextMenuPanel');
        expect(popoverExists).to.eql(false, 'All popovers should be closed');
    });
}
```

Reply:
Updated here https://github.com/elastic/kibana/pull/101316/commits/6c51e92cd5f9ce63fd4b5bcefb38a46e0b7f633e